pack/td.json

```diff
@@ -526,6 +747,75 @@
         "uniqueness": false
     },
     {
+        "advancement_cost": 4,
+        "agenda_points": 2,
+        "code": "13042",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "illustrator": "Ed Mattinian",
+        "keywords": "Security",
+        "pack_code": "td",
+        "position": 42,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "Place 1 agenda counter on Armored Servers when you score it.\n<strong>Hosted agenda counter</strong>: For the remainder of this run, the runner must trash a card from his or her grip as an additional cost to jack out or break a subroutine. Use this ability only during a run.",
+        "title": "Successful Field Test",
+        "type_code": "agenda",
+        "uniqueness": false
+    },
+    {
+        "advancement_cost": 3,
+        "agenda_points": 1,
+        "code": "13043",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "flavor": "\"Anything to anyone...for the right price.\"",
+        "illustrator": "Ed Mattinian",
+        "keywords": "Expansion",
+        "pack_code": "td",
+        "position": 43,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "When you score Illicit Sales, you may take 1 bad publicity. Gain 3[credit] for each bad publicity that you have.",
+        "title": "Illicit Sales",
+        "type_code": "agenda",
+        "uniqueness": false
+    },
+    {
+        "advancement_cost": 5,
+        "agenda_points": 3,
+        "code": "13044",
+        "deck_limit": 3,
+        "faction_code": "weyland-consortium",
+        "illustrator": "Del Borevio",
```

Comment:
I believe the illustrator's name is **Del Borovic** : http://delborovic.tumblr.com/